SpatialGDK/Build/Scripts/DeploymentLauncher.bat

```diff
@@ -0,0 +1,14 @@
+@echo off
+
+set DEPLOYMENT_LAUNCHER_EXE_PATH="%~dp0..\..\..\..\..\Plugins\UnrealGDK\SpatialGDK\Binaries\ThirdParty\Improbable\Programs\DeploymentLauncher\DeploymentLauncher.exe"

```

Comment:
This will break with the unrealgdk as an engine plugin. That's okay for now, as we don't officially support it, but we should probably track this in a ticket - @joshuahuburn ?

Reply:
Made a suggestion on how to fix this, go down 3 directories instead of 5 and you should be good.